external/include/stress_tool/st_perf.h

```diff
@@ -21,9 +21,23 @@
 
 #include <sys/time.h>
 
+#if 0
```

Comment:
If these codes under #if 0 are not needed, they can be removed

Reply:
this will be modified later PR